src/python/grpcio/README.rst

```diff
@@ -6,7 +6,7 @@ Package for gRPC Python.
 Installation
 ------------
 
-gRPC Python is available for Linux, Mac OS X, and Windows running Python 2.7.
+gRPC Python is available for Linux, Mac OS X, and Windows.
```

Comment:
While you are at it, can you `s/Mac OS X/macOS` please? That's the current branding of my favorite desktop operating system.